packages/gitbook/src/components/Header/SpacesDropdown.tsx

```diff
@@ -74,6 +74,7 @@ export function SpacesDropdown(props: {
                         url: getSiteSpaceURL(context, otherSiteSpace),
                     }}
                     active={otherSiteSpace.id === siteSpace.id}
+                    currentSpacePath={siteSpace.path}
```

Comment:
Thinking about the RSC issue, even here impact is minimal; should we have only one `SpacesDropdownMenuItems` component that is client side and iterate over the array of variants; instead of serializing some props N times?